gem/lib/middleman-surfnperf_head/extension.rb

```diff
@@ -0,0 +1,78 @@
+# Require core library
+require 'middleman-core'
+require_relative '../utils'
+# Extension namespace
+module Middleman
+  class MyExtension < Extension
+    option :my_option, 'default', 'An example option'
+
+    def initialize(app, options_hash={}, &block)
+      # Call super to build options from the options_hash
+      super
+
+      # Require libraries only when activated
+      # require 'necessary/library'
+
+      # set up your extension
+      # puts options.my_option
+    end
+
+    def after_configuration
+      # Do something
+    end
+
+    # A Sitemap Manipulator
+    # def manipulate_resource_list(resources)
+    # end
+
+    # module do
+    #   def a_helper
+    #   end
+    # end/
+
+    helpers do
+      def surfnperf_head
+        filePath = SurfNPerf::Utils.gem_libdir + '/middleman-surfnperf_head/_head.html.erb'
```

Comment:
change this to `filePath = File.join(::SurfNPerf::Utils.gem_libdir, '../app/views/surfnperf/_head.html.erb')`